models/users.js

```diff
@@ -165,14 +165,17 @@ const fetchPaginatedUsers = async (query) => {
     const allUsers = [];
 
     snapshot.forEach((doc) => {
-      allUsers.push({
-        id: doc.id,
-        ...doc.data(),
-        phone: undefined,
-        email: undefined,
-        tokens: undefined,
-        chaincode: undefined,
-      });
+      const userData = doc.data();
+      if (!userData.roles.archived) {
+        allUsers.push({
+          id: doc.id,
+          ...doc.data(),
```

Comment:
this won't work db query already has 100 users some of them might be archived, lets say 20, now we are just returning 80 users, expectation was to return 100 unarchived users